### DISCUSSION TITLE
Add support for isl29125, SparkFun rgb light sensor

### DISCUSSION THREAD

**Discussion by rice103** [arendst/Tasmota#10461] [Ideas]
I trying to add support to rgb light sensor that i use in different situation on my home to read device state by catching diagnostic led.
I added XSNS_99 and XI2C_96 (first of free) and feature enable at feature7 (first bit free).
Results is full support but non everytime. Sometime isn't discovered but after many restart (4-5) i can see sensor discovered on webui:
![image](https://user-images.githubusercontent.com/35748699/104048058-f3d1be00-51e2-11eb-9aca-0358bf94836b.png)
Someone help me to get a full support everytime and make a pull request after fix all problem (obviously not in 99'th sensor and 96'th i2c)?
In attach there is all file edited:
[https://github.com/rice103/tasmota_supporting_isl29125](https://github.com/rice103/tasmota_supporting_isl29125)
Thanks,
Rice

**Comment by rice103** [comment]
Please help me

**Comment by rice103** [comment]
nobody can help me?

**Reply by barbudor** [reply]
To help we would need to have the device ?

What have you done to debug ?
You probably should add logs in the detection routine as well as analyzing the I2C transaction with an logic analyzer
